forem/forem
# Remove buffer gem and related code: step 1

## What type of PR is this? (check all applicable)


## Description

This code removes all functionality related to Buffer. Buffer stopped supporting the API we used a while ago, so this was absolute legacy code.

Note: columns and tables will be removed in a follow-up, in this PR they have only been added to `ignored_columns` on the relevant models.

## Related Tickets & Documents

https://github.com/forem/internalEngineering/issues/397

## QA Instructions, Screenshots, Recordings

* All specs should pass
* Click around the articles admin area and make sure things are working
* Click around articles and make sure things are working
 
### UI accessibility concerns?

n/a

## Added tests?

- [X] No, and this is why: only removing code 

## [Forem core team only] How will this change be communicated?
